disable failing DDRGES3 test in LAPACK 3.10 test suite, fixes "Error code from DDRGES3 =    9";
see also https://github.com/OpenMathLib/OpenBLAS/issues/4032
author: Kenneth Hoste (HPC-UGent)
--- OpenBLAS-0.3.23/lapack-netlib/TESTING/dgd.in.orig	2023-09-29 08:05:53.089031858 +0200
+++ OpenBLAS-0.3.23/lapack-netlib/TESTING/dgd.in	2023-09-29 08:08:32.234680735 +0200
@@ -1,6 +1,6 @@
 DGS               Data for the Real Nonsymmetric Schur Form Driver
 5                 Number of matrix dimensions
-6 2 10 12 20 30   Matrix dimensions
+2 10 12 20 30   Matrix dimensions
 1 1 1 2 1         Parameters NB, NBMIN, NXOVER, NS, NBCOL
 10                Threshold for test ratios
 .TRUE.            Put T to test the error exits
